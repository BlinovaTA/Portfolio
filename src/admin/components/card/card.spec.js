import card from "./card.vue";
import { mount } from "@vue/test-utils";

let wrapper;

beforeEach(() => {
  wrapper = mount(card);
});

afterEach(() => {
  wrapper.destroy();
});

describe("Card tests", () => {
  it("card snapshot", () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it("card with title snapshot", async () => {
    await wrapper.setProps({
      title: "title"
    });

    expect(wrapper.element).toMatchSnapshot();
  });

  it("card simple view snapshot", async () => {
    await wrapper.setProps({
      simple: true
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
