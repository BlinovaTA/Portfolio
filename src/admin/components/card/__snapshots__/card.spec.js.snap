// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card tests card simple view snapshot 1`] = `
<div
  class="card-component card_plain"
/>
`;

exports[`Card tests card snapshot 1`] = `
<div
  class="card-component"
>
  <div
    class="header"
  >
    <div
      class="text"
    />
     
  </div>
   
  <div
    class="content"
  />
</div>
`;

exports[`Card tests card with title snapshot 1`] = `
<div
  class="card-component"
>
  <div
    class="header"
  >
    <div
      class="text"
    >
      title
    </div>
     
    <!---->
  </div>
   
  <div
    class="content"
  />
</div>
`;
